### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 12

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#300] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110604](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a4806828-092b-4f37-9913-80ad0da9a49a)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/d7c7afd8-1acb-4272-802c-db5db8ab7afa)

**Comment by Frisbon** [comment]
Ho fatto a caso, non ho capito, ma mi é venuto

![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/86427a98-1022-44c7-b948-aa9fa3d527bb)
